hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -477,10 +485,10 @@ public void testNamespaceExists() throws TException {
 
     catalog.createNamespace(namespace, meta);
 
-    Assert.assertTrue("Should true to namespace exist", catalog.namespaceExists(namespace));
-    Assert.assertTrue(
-        "Should false to namespace doesn't exist",
-        !catalog.namespaceExists(Namespace.of("db2", "db2", "ns2")));
+    assertThat(catalog.namespaceExists(namespace)).as("Should true to namespace exist").isTrue();
+    assertThat(!catalog.namespaceExists(Namespace.of("db2", "db2", "ns2")))
```

Comment:
this is confusing. I'd rather rewrite this to `assertThat(catalog.namespaceExists(Namespace.of("db2", "db2", "ns2")))).isFalse()`

Reply:
Yeah, changed